src/Book.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Objects;
+
+public class Book {
+
+    private final String title;
+    private String Author;
```

Comment:
```suggestion
    private String author;
```
названия полей, переменных, методов нужно писать с маленькой буквы

Reply:
это должен быть автор, а не строка